templates/docs/examples/patterns/lists/lists-dividers-all.html

```diff
@@ -0,0 +1,35 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Lists / All variations with horizontal divider{% endblock %}
+
+{% block standalone_css %}patterns_lists{% endblock %}
+
+{% block content %}
+<ul class="p-list--divided">
+  <li class="p-list__item is-dotted">
+    Balance compute load in the cloud
+    <ul class="p-list--divided">
+      <li class="p-list__item is-dotted">Balance compute load in the cloud</li>
+    </ul>
+  </li>
+  <li class="p-list__item is-ticked">
+    Balance compute load in the cloud
+    <ul class="p-list--divided">
+      <li class="p-list__item is-ticked">Balance compute load in the cloud</li>
+    </ul>
+  </li>
+  <li class="p-list__item">
+    Balance compute load in the cloud
+    <ul class="p-list--divided">
+      <li class="p-list__item">Balance compute load in the cloud</li>
+    </ul>
+  </li>
+</ul>
+<ol class="p-list--divided p-list--nested-counter">
```

Comment:
Is the `p-list--nested-counter` needed here? It seems that without it the counter doesn't change.

Also, this seems a bit inconsistent with current lists (but maybe we want that).
Right now if you have nested `<ol>`, you will get:

```
1. asfasdf
2. asdfasdf
    1. sdffd
    2. sdfsfd
3. sadfafs
```

and you need the `p-list--nested-counter` to achieve:
```
1. asfasdf
2. asdfasdf
    2.1. sdffd
    2.2. sdfsfd
3. sadfafs
```

with new list it seems that the "nested" style is default, you don't need another class name. But I guess it's better to have it by default and not via additional class name, as it's probably better this way.

Reply:
So, I think that we can just remove nested counter class name from this example as it seems it's not doing anything.

Reply:
Yeah the new ordered list has the nested behaviour by default. Is that ok you think?

Reply:
Yes, let's keep it this way.

Reply:
```suggestion
<ol class="p-list--divided">
```